library/alignment.h

```diff
@@ -15,13 +15,7 @@
 #include <string.h>
 #include <stdlib.h>
 
-#if defined(__GNUC__) && !defined(__ARMCC_VERSION) && !defined(__clang__) \
-    && !defined(__llvm__) && !defined(__INTEL_COMPILER)
-/* Defined if the compiler really is gcc and not clang, etc */
-#define MBEDTLS_COMPILER_IS_GCC
-#define MBEDTLS_GCC_VERSION \
-    (__GNUC__ * 10000 + __GNUC_MINOR__ * 100 + __GNUC_PATCHLEVEL__)
-#endif
+#include "mbedtls/build_info.h"
```

Comment:
Hmm, `alignment.h` is in `library/`, so it is internal, so generally shouldn't be included directly (certainly not externally - it's not part of the public API)

`alignment.h` is included by `common.h` after it includes `build_info.h`, so is this line here necessary?

Reply:
It's not strictly neccessary - only `common.h` includes it. No objection to removing it.